hubblestack/modules/network.py

```diff
@@ -1560,7 +1561,10 @@ def connect(host, port=None, **kwargs):
         if proto == "udp":
             # Generate a random string of a
             # decent size to test UDP connection
-            md5h = hashlib.md5()
+            if isFipsEnabled:
+                md5h = hashlib.md5(usedforsecurity=False)
+            else:
+                md5h = hashlib.md5()
```

Comment:
Need to make this change at https://github.com/hubblestack/hubble/blob/develop/hubblestack/modules/nebula_osquery.py#L1232
also

Reply:
Good catch Mudit